apps/docs/src/theme.css

```diff
@@ -73,6 +73,27 @@
   justify-content: center;
   font-size: 0.9rem
 }
+.componentExamplesWrapper {
+  background-color: #fafafa;
+  border-radius: 6px;
+  border: 1px solid rgb(233, 236, 239);
+}
+.componentExamplesTabPanel {
```

Comment:
I'm not sure if it would benefit of having a min-height so that it does not jump in size when switching tabs. What do you think?